index.html

```diff
@@ -0,0 +1,292 @@
+              of sharing & opening.
+            </p>
+          </div>
+        </div>
+
+        <div class="row m-1 mt-2 align-items-center main-background">
+          <div class="col">
+            <i class="fas fa-snowflake"></i>
+          </div>
+          <div class="col">
+            <p class="main-text">Winter</p>
+          </div>
+          <div class="col-6">
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing & opening.
+            </p>
+          </div>
+        </div>
+
+        <div class="row m-1 mt-2 align-items-center main-background">
+          <div class="col">
+            <i class="fas fa-industry"></i>
+          </div>
+          <div class="col">
+            <p class="main-text">Factory</p>
+          </div>
+          <div class="col-6">
+            <p>
+              Listen to the speakers from various countries about the messages
+              of sharing & opening.
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <div class="ticket-button-container">
+        <button class="ticket-button p-4 m-5" type="button">
+          Get a Ticket Now
+        </button>
+      </div>
+    </section>
+
+    <section id="home-desktop">
+      <h4 class="text-center text-white pt-5">Main Program</h4>
+      <hr style="width: 10%; height: 3px; color: red; margin-left: 45%" />
+
+      <div class="events-container text-white">
+<div class="events">
+  <i class="fas fa-ticket-alt"></i>
+  <h3 class="main-text pt-2">Tickets</h3>
+  <p>
+    Listen to the speakers from various countries about the messages
+    of sharing & opening.
+  </p>
+</div>
+
+<div class="events">
+  <i class="fas fa-compass"></i>
+  <h3 class="main-text pt-2">Safari</h3>
+  <p>
+    Listen to the speakers from various countries about the messages
+    of sharing & opening.
+  </p>
+</div>
+
+<div class="events">
+  <i class="fas fa-building"></i>
+  <h3 class="main-text pt-2">City</h3>
+  <p>
+    Listen to the speakers from various countries about the messages
+    of sharing & opening.
+  </p>
+</div>
+
+<div class="events">
+  <i class="fas fa-snowflake"></i>
+  <h3 class="main-text pt-2">Winter</h3>
+  <p>
+    Listen to the speakers from various countries about the messages
+    of sharing & opening.
+  </p>
+</div>
+
+<div class="events">
+  <i class="fas fa-industry"></i>
+  <h3 class="main-text pt-2">Factory</h3>
+  <p>
+    Listen to the speakers from various countries about the messages
+    of sharing & opening.
+  </p>
+</div>
+      </div>
+
+      <div class="pb-5">
+      <h4 class="text-center text-white pt-5">VIEW ALL EVENTS</h4>
+      <hr style="width: 10%; height: 2px; color: white; margin-left: 45%" />
+      </div>
+    </section>
+
+    <section id="speakers"></section>
+    <section id="speakers-desktop"></section>
+
+    <section id="partners" class="text-center p-3 mt-4">
```

Comment:
The partners section requires more padding on the top and bottom. I would also suggest the use of images instead of `<p>` tags

Reply:
:heavy_check_mark: